packages/nimble-components/src/table/models/table-layout-manager.ts

```diff
@@ -77,8 +83,12 @@ export class TableLayoutManager<TData extends TableRecord> {
         this.initialTableScrollableMinWidth = this.table.tableScrollableMinWidth;
         this.initialColumnTotalWidth = this.getTotalColumnFixedWidth();
         this.isColumnBeingSized = true;
-        document.addEventListener('mousemove', this.onDividerMouseMove);
-        document.addEventListener('mouseup', this.onDividerMouseUp);
+        // pointerId of -1 indicates source was synthetic PointerEvent: https://w3c.github.io/pointerevents/#dom-pointerevent-pointerid
+        if (pointerId !== -1) {
```

Comment:
In tests where we create synthetic `PointerEvent`s, we don't have a real, active pointer whose ID we can put in the event. Passing `1` avoided an error on Chrome, but not Firefox. The best solution I can come up with is to use `-1` and treat that as a signal to skip calling `setPointerCapture`.